python/ray/serve/autoscaling_policy.py

```diff
@@ -119,8 +124,15 @@ def __init__(self, config: AutoscalingConfig):
         self.decision_counter = 0
 
     def get_decision_num_replicas(
-        self, current_num_ongoing_requests: List[float], curr_target_num_replicas: int
+        self,
+        current_num_ongoing_requests: List[float],
+        curr_target_num_replicas: int,
+        current_handle_queued_queries: float,
     ) -> int:
+
+        if current_handle_queued_queries > 0 and len(current_num_ongoing_requests) == 0:
+            return max(1, curr_target_num_replicas)
```

Comment:
how do we deal with the edge case of the ongoing_requests is like 0.01 due to window averages?

Reply:
When ongoing_requests is bigger than 0, this if check won't take effect, the desired replicas will be calculated from `calculate_desired_num_replicas` (old behavior, this pr is not going to change the old behavior)

Reply:
can you document what does this section do in code?

Reply:
also should this be nested with condition `len(current_num_ongoing_requests) == 0:` below for readability? so the logic goes
```
if len(current_num_ongoing_requests) == 0:
    if current_handle_queued_queries > 0: # scale up scenario
        return max(...)
    return curr_target_num_replicas
```